nixnet/session.py

```diff
@@ -22,22 +22,76 @@
     "FrameOutStreamSession",
     "FrameInQueuedSession",
     "FrameOutQueuedSession",
+    "FrameInSinglePointSession",
+    "FrameOutSinglePointSession",
     "SignalInSinglePointSession",
     "SignalOutSinglePointSession"]
 
 
 class SessionBase(object):
+    """Session base object.
```

Comment:
This should not be public and doesn't need the same level of documentation; we should move this base class, maybe to `_session/base.py`

Alternatively, if this is truly useful for all leaf classes (including conversion), then it might be kept, just remove the reference to this being a session base class.